eth/backend.go

```diff
@@ -89,6 +90,8 @@ type Ethereum struct {
 
 	engine consensus.Engine
 
+	APIBackend *EthAPIBackend
```

Comment:
We do not want to reintroduce EthAPIBackend here. All RPC communication is happening via RPC daemon

Reply:
What should I use an alternative programmatically?

Reply:
At the moment, the code goes into `cmd/rpcdaemon/commands`, but I would leave it for a separate PR. We need to see what extra RPC requests need to be implemented, and how necessarily data can be delivered to the RPC Daemon. Due to architecture of separability of RPC Daemon and Erigon, there needs to be a way of delivering data both local and remotely